pack.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
   - issues
   - ticket management
   - project management
-version: 3.0.1
+version: 3.0.2
```

Comment:
Feels like this might be a minor version bump given the input to a commonly used action is being changed, so probably should be `3.1.0` here

Reply:
Updated.